Bottom: f6ccd17e23ac399cd4c50553197c40209aeee0fa
Top:    291014435e906551bc43e1720027865014cbc7f6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 13:34:07 +0100

Refresh of add-support-for-a-poured-objec-I

---

diff --git a/src/search.c b/src/search.c
index 19216d0..9e8c005 100644
--- a/src/search.c
+++ b/src/search.c
@@ -477,7 +477,7 @@ pour_callback (const BoxType * box, void *cl)
 /* ---------------------------------------------------------------------------
  * searches a pour on the SearchLayer 
  */
-static Boolean
+static bool
 SearchPourByLocation (int locked, LayerTypePtr * Layer,
                       PourTypePtr * Pour, PourTypePtr * Dummy)
 {
